packages/cli/src/templates/project/package-json.ts

```diff
@@ -1,15 +1,18 @@
+// New projects use the same Booster version as the installed CLI
+const VERSION = require('../../../package.json').version
+
 export const template = `{
   "name": "{{{projectName}}}",
   "description": "{{{description}}}",
   "version": "{{{version}}}",
   "author": "{{{author}}}",
   "dependencies": {
-    "@boostercloud/framework-core": "^0.5.0",
-    "@boostercloud/framework-types": "^0.5.0",
+    "@boostercloud/framework-core": "^${VERSION}",
+    "@boostercloud/framework-types": "^${VERSION}",
     "{{{providerPackageName}}}": "*"
   },
   "devDependencies": {
-    "@boostercloud/cli": "^0.5.0",
+    "@boostercloud/cli": "^${VERSION}",
```

Comment:
Not the versions in the project template are dynamically filled with the current version of the CLI.